### PR TITLE
EDX Ecommerce Arabic Language Translations Updated

### PR DESCRIPTION
**Pull request by asadmanzoor93**
**Description:**  This PR adds missing translations for Arabic language support in Edly Ecommerce

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-1604

**Merge checklist:**

- [ ] All reviewers approved
- [ ] Commits are (reasonably) squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Note:** Please add screenshots for any viewable change.
Screenshot 2020-07-09 at 11 15 05 AM
Screenshot 2020-07-09 at 11 15 11 AM
Screenshot 2020-07-09 at 11 15 36 AM
Screenshot 2020-07-09 at 11 15 46 AM

### CODE REVIEW

File: ecommerce/conf/locale/ar/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -496,4 +509,6 @@ msgstr "حدث خطأ"
 msgid ""
 "Failed to process refund #%(refund_id)s: %(error)s. Please try again, or "
 "contact the E-Commerce Development Team."
-msgstr "تعذّرت معالجة طلب إعادة المبلغ ذا الرقم #%(refund_id)s: %(error)s. يُرجى إعادة المحاولة أو الاتصال بفريق تطوير التجارة الإلكترونية."
+msgstr ""
+"فشلت معالجة رد الأموال #٪ (refund_id) s:٪ (خطأ) s. يرجى المحاولة مرة أخرى ، أو"
```
Comment (zubair-arbi) [inline ecommerce/conf/locale/ar/LC_MESSAGES/djangojs.po:513]: Please revert this (keep the old changes). The new translation is not correct (placeholders are not added correctly).

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -643,13 +653,13 @@ msgstr "غير نشط"
...
 msgid "No SKUs provided."
-msgstr ""
+msgstr "لم يتم توفير وحدات SKU."
 
 #: ecommerce/extensions/api/v2/views/baskets.py:373
 #: ecommerce/extensions/basket/views.py:61
 #, python-brace-format
 msgid "Products with SKU(s) [{skus}] do not exist."
-msgstr ""
+msgstr "المنتجات التي تحتوي على SKU (s) [{skus}] غير موجودة."
```
Comment (zubair-arbi) [inline ecommerce/conf/locale/ar/LC_MESSAGES/django.po:662]: `SKU(s)`

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -742,7 +760,7 @@ msgstr ""
 #: ecommerce/extensions/basket/views.py:363
 #, python-brace-format
 msgid "Could not apply the code '{code}'; it requires data sharing consent."
-msgstr ""
+msgstr "تعذر تطبيق الرمز '{code}' ؛ يتطلب موافقة مشاركة البيانات."
```
Comment (zubair-arbi) [inline ecommerce/conf/locale/ar/LC_MESSAGES/django.po:763]: Place of `;` is correct?

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -850,29 +868,29 @@ msgstr "عذرًا، لقد فشلنا في استرجاع بيانات التس
...
 
 #: ecommerce/extensions/offer/models.py:287
 msgid "UUID for an EnterpriseCustomer from the Enterprise Service."
-msgstr ""
+msgstr "UUID لعميل EnterpriseCustomer من خدمة Enterprise."
 
 #: ecommerce/extensions/offer/models.py:389
 msgid "EnterpriseCustomer UUID"
-msgstr ""
+msgstr "UCID EnterpriseCustomer"
```
Comment (zubair-arbi) [inline ecommerce/conf/locale/ar/LC_MESSAGES/django.po:879]: remove this

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -850,29 +868,29 @@ msgstr "عذرًا، لقد فشلنا في استرجاع بيانات التس
...
 
 #: ecommerce/extensions/offer/models.py:396
 msgid "EnterpriseCustomer Name"
-msgstr ""
+msgstr "اسم العميل"
 
 #: ecommerce/extensions/offer/models.py:401
 msgid "EnterpriseCustomerCatalog UUID"
-msgstr ""
+msgstr "UCID EnterpriseCustomerCatalog"
```
Comment (zubair-arbi) [inline ecommerce/conf/locale/ar/LC_MESSAGES/django.po:887]: remove this

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -1834,7 +1862,7 @@ msgstr ""
 
 #: ecommerce/templates/customer/emails/commtype_order_with_csv_body.txt:14
 msgid "For more information and assistance, contact info@edx.org."
-msgstr ""
+msgstr "لمزيد من المعلومات والمساعدة ، اتصل info@edx.org."
```
Comment (zubair-arbi) [inline ecommerce/conf/locale/ar/LC_MESSAGES/django.po:1865]: Remove this.

Where is this message showing? We might need to change this message due to `info@edx.org` link if users will see this message.

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -1848,7 +1876,7 @@ msgstr "تسجيل الخروج"
 #: ecommerce/templates/edx/checkout/receipt.html:11
 #, python-format
 msgid "Receipt for %(order_number)s"
-msgstr ""
+msgstr "إيصال %(order_number) s"
```
Comment (zubair-arbi) [inline ecommerce/conf/locale/ar/LC_MESSAGES/django.po:1879]: `%(order_number) s` or `%(order_number)s`?

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -2482,6 +2520,10 @@ msgid ""
 "information about options for clearing a match. Your account will be "
 "suspended until this matter is resolved satisfactorily."
 msgstr ""
+"إذا كانت لديك أسئلة بخصوص مسح مباراة ، فيرجى الاتصال"
```
Comment (zubair-arbi) [inline ecommerce/conf/locale/ar/LC_MESSAGES/django.po:2523]: remove this

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po (hunk 3 above)
Comment (asadmanzoor93) [reply ecommerce/conf/locale/ar/LC_MESSAGES/django.po:763]: I have converted this using Google translate.

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po (hunk 6 above)
Comment (asadmanzoor93) [reply ecommerce/conf/locale/ar/LC_MESSAGES/django.po:1865]: @zubair-arbi this message is part of Email send on purchasing course. ecommerce/ecommerce/templates/customer/emails/commtype_order_with_csv_body.txt

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po (hunk 3 above)
Comment (asadmanzoor93) [reply ecommerce/conf/locale/ar/LC_MESSAGES/django.po:763]: @zubair-arbi in code-base it showing correct place of `;`
Screenshot 2020-07-09 at 2 13 30 PM

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po (hunk 8 above)
Comment (asadmanzoor93) [reply ecommerce/conf/locale/ar/LC_MESSAGES/django.po]: Done

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po (hunk 7 above)
Comment (asadmanzoor93) [reply ecommerce/conf/locale/ar/LC_MESSAGES/django.po]: Space added by google translator. now removed.

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po (hunk 3 above)
Comment (zubair-arbi) [reply ecommerce/conf/locale/ar/LC_MESSAGES/django.po:763]: Google translations are not accurate sometimes. We should not add translations for which we are not sure (regarding format). Also this string is for enterprise flow which we are not using.

Remove this translation.

File: ecommerce/conf/locale/ar/LC_MESSAGES/djangojs.po (hunk 1 above)
Comment (asadmanzoor93) [reply ecommerce/conf/locale/ar/LC_MESSAGES/djangojs.po]: Done

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po (hunk 2 above)
Comment (asadmanzoor93) [reply ecommerce/conf/locale/ar/LC_MESSAGES/django.po]: Done

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po (hunk 4 above)
Comment (asadmanzoor93) [reply ecommerce/conf/locale/ar/LC_MESSAGES/django.po]: Done

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po (hunk 5 above)
Comment (asadmanzoor93) [reply ecommerce/conf/locale/ar/LC_MESSAGES/django.po]: Done

File: ecommerce/conf/locale/ar/LC_MESSAGES/django.po (hunk 6 above)
Comment (asadmanzoor93) [reply ecommerce/conf/locale/ar/LC_MESSAGES/django.po]: Done

**Pull request merged**